Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```diff
@@ -65,24 +65,21 @@
       - id: ClothingNeckCloakHop
       - id: ClothingHeadHatHopcap
       - id: HoPIDCard
-        prob: 0.9
       - id: ClothingHeadsetAltCommand
       - id: BoxPDA
       - id: BoxID
       - id: BoxHeadset
-    #  - id: WeaponDisabler
       - id: ClothingShoesColorBlack
         prob: 0.7
       - id: PlushieLizard
         prob: 0.1
       - id: CigarGoldCase
         prob: 0.10
-        # Fuck the HoP they don't deserve fucking cigars.
-    #  - id: DoorRemoteService
```

Comment:
I probably should've done this quite some time ago, but change this to DoorRemoteCommand and add it back.

Reply:
righto, it's done